.github/workflows/windows_build_test.yml

```diff
@@ -23,11 +23,13 @@ jobs:
           auto-activate-base: true
           activate-environment: ""
 
+      # yaml-cpp package has conflicting gtest headers and is not needed for this project
       - name: Conda dependencies
         shell:  bash -l {0}
         run: |
           conda install curl eigen
           conda install -c conda-forge hdf5=1.10.6
+          conda remove -y yaml-cpp
```

Comment:
interesting I wonder how this one is brought in to the environment.

Reply:
conda install with ```--no-deps ```would perhas be another option

Reply:
We are using a standard miniconda environment as the windows image, so it probably is just there from that process

Reply:
... so it's only really relevant for CI, this this is a pretty good solution, I think